caom2pipe/tests/data/test_cache.yml

```diff
@@ -0,0 +1,14 @@
+value_repair:
+  observation.type:
+    dark: DARK
+    Dark: DARK
+  observation.proposal.pi_name:
+    jjk: JJ Kavelaars
+  plane.meta_release:
+    any: 2000-01-01T00:00:00.000
```

Comment:
Can't this be covered by blueprint config file? I can't remember.

Reply:
The use of a config file with the blueprint was for legacy instances - none of *2caom2 use one. Did I mis-understand completely how the blueprint was supposed to be used?